Experiments_toolkit.py

```diff
@@ -0,0 +1,190 @@
+import tensorflow as tf
+
+
+def create_train_test_validation(positives, negatives, percent_train, percent_test, percent_validation):
+    """
+        Description: creates a training, testing, and validation dictionary for binary data
+
+        Args:
+            positives -- np_array (List of positive samples as numpy arrays)
+            negatives -- np_array (List of negative samples as numpy arrays)
+            percent_train -- double ([0-1] Fraction of data to go to training set)
+            percent_test -- double (0-1] Fraction of data to go to testing set)
+            percent_validation -- double ([0-1] Fraction of data to go to validation set)
+
+        Return:
+            train -- dict (Dictionary with percent_train of original dictionary)
+            test -- dict (Dictionary with percent_test of original dictionary)
+            validation -- dict (Dictionary with percent_validation of original dictionary)
+        """
+    train = {"positives": [], "negatives": []}
+    test = {"positives": [], "negatives": []}
+    validation = {"positives": [], "negatives": []}
+
+    if (percent_train + percent_test + percent_validation != 1):
+        print("ERROR: percentages don't add to 1")
+
+    train, test, validation = splitting_up_data(
+        positives, "positives", train, percent_train, test, percent_test, validation, percent_validation)
+    train, test, validation = splitting_up_data(
+        negatives, "negatives", train, percent_train, test, percent_test, validation, percent_validation)
+
+    return train, test, validation
+
+
+def splitting_up_data(data, key, train, percent_train, test, percent_test, validation, percent_validation):
+    """
+        Description: Splits up data from dictionary using the given percentages to make a train, test, and validation set
+
+        Args:
+            data -- np array (Array of samples as numpy arrays)
+            key -- String (Key for the in dictionary from which we will split up the data from)
+            train -- dict (Training dictionary )
+            percent_train -- double ([0-1] Percentage of data to go to training set)
+            test -- dict (Testing dictionary )
+            percent_test -- double (0-1] Percentage of data to go to testing set)
+            validation -- dict (Validation dictionary )
+            percent_validation -- double ([0-1] Percentage of data to go to validation set)
+
+        Return:
+            train -- dict (Dictionary with percent_train of list)
+            test -- dict (Dictionary with percent_test of list)
+            validation -- dict (Dictionary with percent_validation of list)
+        """
+    length = len(data)
+    train_index = int(length * percent_train)
+    test_index = int(length * percent_test)
+    train[key] = data[0:train_index]
+    test[key] = data[train_index:train_index + test_index]
+    validation[key] = data[train_index + test_index:]
+    return train, test, validation
```

Comment:
Can you add randomization here as well to make sure the result isn't dependent on the order?